phantom-dsl/src/test/scala/com/websudos/phantom/tables/OptionalPrimitivesCassandra22.scala

```diff
@@ -37,7 +37,8 @@ import com.websudos.util.testing._
 case class OptionalPrimitiveCassandra22(
                                          pkey: String,
                                          short: Option[Short],
-                                         byte: Option[Byte]
+                                         byte: Option[Byte],
```

Comment:
@mmatloka Have a look at [this](http://stackoverflow.com/questions/26880677/intellij-scala-plugins-case-class-indentation-is-absurd) for details.